### PR TITLE
AVIARY-3273 - Share links accessible with tab

### PR DESCRIPTION
**Pull request by rubab**

### CODE REVIEW

File: app/assets/javascripts/accessibility.js [hunk 1]
```diff
@@ -36,5 +36,19 @@ function Accessibility() {
             var timerId = setInterval(adjustLocation, 10);
             setTimeout(function() {clearInterval(timerId)}, 500);
         });
+
+        tabChanges();
+    }
+
+    const tabChanges = () => {
+        document_level_binding_element('.vjs-big-play-button', 'focus', function () {
+            console.log("from file")
```
Comment (furqanwasi) [inline app/assets/javascripts/accessibility.js]: same here as private

File: app/assets/javascripts/accessibility.js [hunk 2]
```diff
@@ -36,5 +36,19 @@ function Accessibility() {
...
+            $(".shareHolder").css({"visibility": "visible", "opacity": 1});
+        });
+
+        $(window).on('load', function () {
+            console.log("from file on load")
```
Comment (furqanwasi) [inline app/assets/javascripts/accessibility.js]: same here as private

**Review by furqanwasi — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/accessibility.js (hunk 2 above)
Comment (rubab) [reply app/assets/javascripts/accessibility.js]: console removed

File: app/assets/javascripts/accessibility.js (hunk 1 above)
Comment (rubab) [reply app/assets/javascripts/accessibility.js]: console removed

**Review by furqanwasi — APPROVED** [review]

**Pull request merged**